wpical/src/main/native/cpp/WPIcal.cpp

```diff
@@ -226,6 +227,27 @@ static void DisplayGui() {
   static double imagerWidth = 1920;
   static double imagerHeight = 1080;
 
+  static constexpr const char* charucoDictNames[] = {
```

Comment:
Instead of two separate arrays, you could do arrays of [`std::pair`](https://en.cppreference.com/w/cpp/utility/pair.html)s.
```cpp
static constexpr std::pair<const char*, int> charucoDicts[] = {
  { "DICT_4X4_50", DICT_4X4_50 },
  // etc.
};

// Get a name
charucoDicts[i].first
// Get an enum value
charucoDicts[i].second
```
If you don't like how unclear `first` and `second` are, you can define your own struct:
```cpp
struct CharucoDict {
  const char* name;
  int enumValue;
};

static constexpr CharucoDict charucoDicts[] = {
  { "DICT_4X4_50", DICT_4X4_50 },
  // etc
};

// Get a name
charucoDicts[i].name
// Get an enum value
charucoDicts[i].enumValue
```
Syntax might be slightly off, but hopefully you get the idea.

Reply:
Thanks for the suggestion. Implemented in my latest commit :)